messages.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016 Qiang Xue. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package validation
+
+var messages = map[string]string{
```

Comment:
I think we should get rid of this map. It doesn't provide extra value while it requires us to type the key strings in multiple places, which could cause typo issues.

Reply:
Ok, I remove it, but if users want to see a list of keys to translate, by this map they can simply detect what keys should translate.

Reply:
Yeah, I understood your thought when I saw this. I have created I18N libraries and I18N-enabled applications before. To determine which messages need to be translated, we created simple regex-based code scanners which can easily find all messages that need translation from a huge code base.  I know other I18N systems also support dynamic detection of I18N messages. So I think we don't need to worry too much about finding out what messages need to be translated.

Reply:
You are right